### DISCUSSION TITLE
I can't get my Student pack

### DISCUSSION THREAD

**Discussion by khaledelsayed12** [community/community#65714] [GitHub Education]
### Select Topic Area

General

### Body

I'm really need your support .. I attached my college ID to get the student pack for the fourth times but I didn't get any mails for now and I really need it . I don't understand where the problem is .. I'm student in Commerce In Zagazig university in Egypt . and thank you

**Comment by yalinn** [comment]
when did you apply for student pack (for the first time)? it's a process that can take time. 
and remember the email you use in apply have to be registered in [settings/emails](https://github.com/settings/emails)
if there is nothing wrong, you should wait a little.

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @khaledelsayed12 , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category.  I've gone ahead and moved it for you. Good luck!
